packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -136,10 +131,7 @@ class DevFSFileContent extends DevFSContent {
   }
 
   @override
-  Future<List<int>> contentsAsBytes() => _getFile().readAsBytes();
-
-  @override
-  Stream<List<int>> contentsAsStream() => _getFile().openRead();
+  List<int> contentsAsBytes() => _getFile().readAsBytesSync();
```

Comment:
Uint8List?

Reply:
I've moved the `.cast<int>()` to the body of this getter instead of in the encoded version. We can't update the return type until the change lands, but the cast makes it forward compatible